app/src/main/res/layout/fragment_home.xml

```diff
@@ -16,4 +16,20 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/log_out_label" />
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/helpMessageTIET"
+            android:layout_width="200dp"
```

Comment:
Usually `TextInputEditText` has `match_parent` as width, and `TextInputLayout` has a fixed width (or 0dp/match_parent)